web/hooks/use-user-bets.ts

```diff
@@ -59,3 +61,13 @@ export const useGetUserBetContractIds = (userId: string | undefined) => {
 
   return contractIds
 }
+
+export const useUserSwipes = () => {
+  const user = useUser()
+  const [swipes, setSwipes] = useState<string[]>()
```

Comment:
A simple improvement would be to switch `useState` for

```
usePersistentState<string[] | undefined>(undefined, {
    store: inMemoryStore(),
    key: `user-swipes-${user.id}`,
  })
```
Which would keep the data loaded in case they navigate back.